### DISCUSSION TITLE
Research activities at Life Itself and a Labs "Research Community" [rfc]

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#261] [Research]
**MOVED to notebook https://lifeitself.org/notes/research-at-life-itself-2023**

**If you are interested in participating just drop a comment in the thread below - we welcome new participants!**

# Summary

Research is a central activity at Life Itself.  Research has the broad meaning of "systematic investigation". It includes both general inquiry and the stricter, academic meaning of using standard methods and seeking original results.

Research takes place at Labs in two distinct and related spaces: a broader and informal "research community" and a narrower, more formal Labs team.

Research focuses on ontology and culturology i.e. the study of inner development and cultural evolution and how they can be cultivated.

Implications and details such as work plan for 2023 are in progress and up for discussion!

# Proposal

## Introduction

Question: What research should we do at Life Itself and how should we organize it?

- What do we mean by research?
- What topics do we focus on?
- Who is involved?
- How does more formal e.g. funded or academic research fit in?

## Research is a central activity at Life Itself. 

Research has the broad meaning of "systematic investigation". It includes both general inquiry and the stricter, academic meaning of using standard methods and seeking original results.

### Research takes places at Labs

Research takes place at Labs in two distinct connected areas:

- Labs research "community": broader, informal research community 
- Labs itself: a smaller, more formal team doing more formal research e.g. funded and/or academic style

<img src="https://user-images.githubusercontent.com/180658/212433169-9b72a932-9cc4-48b9-a34e-65e9662b7678.png" height="400" />

[[Source](https://app.excalidraw.com/l/9u8crB2ZmUo/A2ZrzcbF90k)]

### Research focuses on ontology and culturology from a rigorous, practical perspective (though ultimately any topic is welcome)

Ultimately any topic is welcome that has a plausible connection with a wiser weller world.

However, we do have specific themes and a certain flavour to our approach.

<img src="https://user-images.githubusercontent.com/180658/212432819-00475aad-cb94-4fbe-b90f-4357f0a5d394.png" height="500" />

#### Primary theme is ontology and culturology aka inner development and cultural evolution

Primary thematic focus is ontology and culturology i.e. the study ("science") of personal and collective inner development. Aka inner development and cultural evolution.

Specifically their nature and causes i.e. a) the nature of inner development and cultural evolution b) causes and the factors such practices/environments that influence them.

In terms of factors we have a particular interest in **conscious collectives and deliberately development spaces**.

TODO: a note on the duality / inter-twinedness of personal and collective development at multiple levels

We are also interested in the impacts of ontology and culturology, particularly in relation to wellbeing and wisdom/performance at the personal and collective[^1] level.

[^1]: teams, coliving communities, societies etc.

#### Secondary theme: paradigmatic evolution of socio-economic (eco)systems.

A secondary and complementary theme is paradigmatic evolution of socio-economic (eco)systems. For example, a shift from a proprietary digital economy to an open one, or a carbon-based energy system to a post-carbon one.

#### Tertiary focus: politics of paradigmatic social change

A tertiary theme is the politics of paradigmatic change as in collective action and movement building for a foundational shift in our onto-socio-economic systems. This integrates the previous two areas in that paradigmatic shift incorporates a substantive evolution in both our cultural and socio-economic systems. This theme is also a concretizing one where the ideas in the previous two areas turn into concrete policy and practices for transformative change.
 
Within this broad area a specific and active sub-area is **ecosystem mapping and field-building** i.e. delineating a field/ecosystem, identifying actors, weaving them together etc.

#### Flavor: rigorous, open-minded, generous, practical

We also have certain principles & practices that inform our approach (in progress)

- Rigor
- Open-mindedness
- Listening generously (aka steel-manning)
- Practical (Praxis?)

#### Development of the Research Community (in progress)

Development of informal Labs "Research Community" has particular importance.

Existing proposed activities like reading group on stage theory (#186) would take place here.

**Comment by rufuspollock** [comment]
Draft re Life Itself research activities is up for comment and improvement @Liyubov @theo-cox @catherinet1 @lkavanagh30 @boazBF @baouroux @Alex-Goodall @msantolini @nathenf ...

**Reply by msantolini** [reply]
Fantastic summary!

**Reply by baouroux** [reply]
Great overview...very exciting to see these themes pulled together, from culturology through to "politics of paradigmatic social change". 

I'm not sure if this is worth including as a sub-section (might already be anticipated under the ontology section), but I'm thinking of how the evolution of anthropological approaches might be relevant...i.e. making sense of what it is to be a participant observer, what are the limitations of 'well-intentioned' change agents. For example including the observer into the systems we are hoping to make sense of, and being conscious of complexity principles.  Narratives and Sensemaking would be also great to add in somewhere, in relation to this thread. I'm particularly interested in the "Science of Narrative"...(ref. a whole new era in understanding economics through the lens of narratives, as per Robert Schiller's book "Narrative Economics"). 

John Vervaeke's 4 ways of knowing (Procedural, Propositional, Participatory and Perspectival) also might be an interesting set of sub-headings in the "Onto Stuff" to include down the line. I've been playing with how these map onto the Cynefin framework and the Integral Quadrants. 

What innovations (incl AI mapping) might be on the horizon for how all the meta-modern frameworks can be navigated and connected? Managing the potential overwhelm?!

**Reply by rufuspollock** [reply]
@baouroux a great set of rich points.

To pick just one juicy one: mapping the various frameworks e.g. cynefin, integral quadrants etc together would be very useful first exercise - perhaps you could do a short "brown-bag" session presenting these ...

**Comment by rufuspollock** [comment]
As a first step on this, i'm proposing a regular friday meeting at 1430 CET for 1h. People can join for as much or as little as they can. Let me know if you'd like to be added to the calendar invite.

**Reply by Alex-Goodall** [reply]
Please add me, Rufus, but at the moment I'm not available till the evening on Fridays.

**Reply by rufuspollock** [reply]
@Alex-Goodall will do! Can you ping me your preferred email to rufus at life itself dot org.

**Comment by baouroux** [comment]
Happy to do so Rufus...can be just a very quick exercise in seeing just
potentially such sources such as Integral, Cynefin and Vervaeke's Ways Of
Knowing might connect. Reinforcing perhaps the clarity in our sensemaking,
seeing how the different terms/segmentations do relate to each other.

John Oliver

**Reply by Alex-Goodall** [reply]
That would be wonderful, John!

And if you could throw in the 4E's of 4E Cognitive Science (which I think align somehow), you'd be an even bigger hero!

Alex

**Comment by asimong** [comment]
Hi Everyone, Theo pointed me to this conversation into which I hope I can contribute. Among other things I'm very interested in the methodology for developing collective ontologies to support standards, interoperability, communication and collaboration on things that are of shared concern. And, perhaps less obviously, having an IFS-aware (or you could say parts-aware) approach to collaborative dialogue.

As I said to Rufus in November in Bergerac, I'm a great fan of "open" (and "commons") in general, and have experience with interoperability standardization, and am in touch with various others interested in standards in related areas. And higher education and research of course. Looking to find how I can best contribute uniquely to good initiatives. Can we open up higher education? Can we build an open system for research quality?

Also I have an extremely long-term interest in systems to help people find each other.

Some of my recent writings are here: https://wiki.simongrant.org/doku.php/here

Happy to explore collaboration on things that we mutually value.

In principle Friday meetings sound good, and I'd like to be added, just that this Friday I won't be available and not sure about the following 4, but let's see.

Simon

**Reply by Alex-Goodall** [reply]
Hi Simon - I think we have strong overlaps with our interests related to the Ontology theme.

I keep delving into the idea of creating an ontology for this space, and then run away in fear of being sucked into a rabbit hole (or more accurately, a warren).

I originally started this as a way of better organizing the content on the Liminal Learning Portal (http://tllp.org). I got as far as identifying 384 terms (before I even started trying to do it systematically), and attempting to create a classification and structing of them. I could see it was turning into an unjustifiably large project.

Tagging content on tllp is optional, but for my other (main) project (http://Relevant.Education) I think it's essential. 

Given the context of the project ("What you need to know to consciously participate in Humanity's Transtion" - to be accessbile to 16-year olds and above), the value of such an ontology would be to help with creating a "Syllabus Framework" for use as the basis of:

Online educational programs.
Material usable by local groups (e.g. Meetups)
Creating material that can be replicated, re-used and further developed by a community.

ON THE OTHER HAND...
I'm aware of my instincts to do excessive left-brain-focussed tagging and structuring. So perhaps the aim should not be to go as detailed as possible, but to find the smallest ontology that characterises the essence, and dump everything else into a simple glossary (thinking out load here, so time to stop!)

As an aside, I could classify Relevant.Education as a "deliberately developmental program"

**Reply by asimong** [reply]
Hi Alex, great to read your reply!

Having been a teacher/lecturer/educator at various levels over my career, I share some background :-)

The **essence** of what I am suggesting is the methodology: that no individual one of us starts by trying to categorise, classify, or create an ontology; but that we use dialogue practices (cf Bohm; Isaacs; Scharmer) to provide a basis from which schemes can emerge, or maybe be coaxed out.

Your use of the term "deliberately developmental" for me strongly tags Robert Kegan, and I'm a great fan of his developmental schema, and indeed of his predecessor William G Perry.

And then there's Internal Family Systems, which I've just got into... ;-)

Shall we talk sometime? I sense the potential for synergy in co-creation...

**Reply by asimong** [reply]
Oh, and I do like the flavour of your Relevant Education project!

**Reply by Alex-Goodall** [reply]
Yes - let's do that!

I suspect that any methodology is going to be an improvement over my addled brain fighting with a Google Sheet.

I picked up "deliberately developmental" from Rufus' Topic Map, but I'm familiarish with, and a big fan of Kegan.

Be in touch.

Alex

**Comment by baouroux** [comment]
Hi everyone,

Just wanted to pick up on a couple of threads from Joe's presentation last
Friday - thanks Joe for your presentation. Great to see how many themes
connect all our interests.

To bookmark a few links, Joe's presentation reminded me of a very succinct
summary by the artist/computer scientist Jonathan Harris, in that social
media needs to move from:

   - from Disposability to Timelessness
   - from Compression to Deepening
   - from Curation to Creation
   - from Self-Promotion to Self-Awareness

He presents these ideas in the video here: https://youtu.be/mhPjrlPOTG8

Marc also mentioned tapping into another energy in the way social sharing
content is generated and shared. An inquiry for me into this has been the
cornerstone for the past 3 years in moving away from the large-scale
corporate sector text narratives and sensemaking, to the other end of the
spectrum of narratives: 2nd person video portrait witnessing, with the
gallery so far here: www.interiortruth.com.

Which isn't very scalable! But it does tap into the expression by Jung that
'what is most personal, is most universal'....

Silence-based narrative sharing with my research seems also to have many
aspects in common with the Quaker vocal ministry practice through silence*
and Bohmian dialogue.

There are also figures such as Olen Gunnlaugson with the Integral We Space
initiatives...I'm still diving into his writing, including the invocations
they use for their sessions.

So the inquiry for the intimacy for this We Space to indeed scale supported
by the innovations in the platforms is an essential one....and I see your
start-up project Joe pointing towards that!!

All the best,
John

*"*Quakers are known for gathering to worship in silence, and yet they also
gather to hear the many voices of God. Instead of a prepared sermon or
liturgy, Quakers worship through “vocal ministry,” messages offered out of
the silence by those who feel moved to speak.*

John Oliver
Mobile: +33 631 60 2994
Bordeaux, France
www.tyler.world (art practice)
www.interiortruth.com (witnessing practice)
www.collectivenavigator.com (dialogue platform)
*LinkedIn* <https://www.linkedin.com/in/john-oliver-bb01a/>

**Comment by msantolini** [comment]
Putting the [link](https://docs.google.com/presentation/d/1t2fYeOA14PBu-TZ--yTYRdGyQNvqMYIry99K0GOU0c8/edit?usp=sharing) to my slides from today here, just in case.

**Reply by rufuspollock** [reply]
@msantolini i think we could create a note in our digital garden / notebook for this ie. at lifeitself.org/notes/ - i'm happy to help make that happen if you are ok with that.

**Reply by msantolini** [reply]
Yes that could be good. By the way I can't access your link?

<img width="1174" alt="image" src="https://user-images.githubusercontent.com/956597/224548537-db0f728d-470f-4fdb-9cb7-ac9c79c178d8.png">

**Reply by rufuspollock** [reply]
notes does not have an index page atm - we want to fix that 😉  - but it is a functional directory https://lifeitself.org/notes/research-at-life-itself-2023

I imagined you might create a note in the vault like `notes/research-meeting-2023-03-10` which would be live at https://lifeitself.org/notes/research-meeting-2023-03-10

**Reply by msantolini** [reply]
Got it. Will be a bit hectic this week with my trip to Geneva, might actually miss Friday's meeting, it's right in the middle of the conference.. 
Just to understand, you imagine a note with some embedding of the slides? (if that's even feasible in obsidian?)

**Comment by baouroux** [comment]
Thanks Marc, super presentation, loved the way you charted the evolution of
your reading and influences.

More as a bookmark to myself as to a question I'd love to follow up on, is
what is being modeled on the components of the edges (the connectors) vs.
the nodes of those network mapping?

I went quite deep in researching the art of data visualisation, and really
enjoyed the topic... A lot going on there in terms of engaging with the
aesthetic processes of the reader and the cognitive aspects. How data is
used, shared, owned

Talking of aesthetics and mapping, I attach a niche paper that connects
with the arts... An oblique connection to the subjects above, but perhaps
of some interest!

**Reply by Liyubov** [reply]
@msantolini 
thanks!
would be great to know more about what you are working on more with Boaz :) 
And I also keep the link to neurosystemics website (residency from Boaz in March 2023) https://neurosystemics.org/
